tests/rules/test_StorageEncryptedRule.py

```diff
@@ -0,0 +1,88 @@
+import pytest
+
+from cfripper.model.enums import RuleGranularity, RuleMode, RuleRisk
+from cfripper.model.result import Failure
+from cfripper.rules.storage_encrypted_rule import StorageEncryptedRule
+from tests.utils import get_cfmodel_from
+
+
+def test_storage_encrypted_rule_valid_results():
+    rule = StorageEncryptedRule(None)
+    model = get_cfmodel_from("rules/StorageEncryptedRule/encrypted_db_resource.yml")
+    resolved_model = model.resolve()
+    result = rule.invoke(resolved_model)
+
+    assert result.valid
+    assert result.failures == []
+
+
+def test_rule_not_invoked_for_aurora():
```

Comment:
```suggestion
def test_rule_not_failing_for_aurora():
```
The rule is always invoked 😅